### ISSUE TITLE
Why not vue-foundation

### ISSUE THREAD

**Issue by ncoden** [Nee-V/NeeV#18]
Why "Neev" instead of "vue-foundation" ?

**Comment by codetheorist** [comment]
So I tried reaching out to Vue Foundation author about becoming a co-maintained but received no response.

I was going to start submitting PRs but the amount of code that I wanted to contribute to the project, from my perspective, was not really feasible submitting PRs constantly. I also wondered about time getting the PRs merged due to the maintainer not responding to my request.

I would also like for this project to be more than a couple of semi-functional components, I would also like it to include templates for Vue-CLI tool, SSR and more. 

The actual name was chosen by @IamManchanda and means Foundation or Root, with the capital V at the end for Vue.

**Comment by IamManchanda** [comment]
> Why "Neev"  instead of "vue-foundation" ?
Yeah @ncoden i asked the same question to him and i think he is right .... 

Also, I have tried to reach out also in the past but with no luck! I think this is a good move!

**Comment by IamManchanda** [comment]
Yeah NeeV means base or Foundation!
